include/mata/nfa/delta.hh

```diff
@@ -440,6 +457,32 @@ public:
      * @return An iterator to @c SymbolPost with epsilon symbol. End iterator when there are no epsilon transitions.
      */
     static StatePost::const_iterator epsilon_symbol_posts(const StatePost& state_post, Symbol epsilon = EPSILON);
+
+    /**
+     * @brief Expand alphabet by symbols from this delta to given alphabet.
```

Comment:
alphabet = given alphabet = parameter was not clear

Reply:
Fixed.